README.md

```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
 - Named as README (case-insensitive): `docs/Guides/README.mdx`
 - Same name as the parent folder: `docs/Guides/Guides.md`
 
-#### Referencing a page
+### Markdown Links and URLs
+
+The URL of a page will be composed of the file path to the page in context from the current file. All references to a another documentation page must end with the `.md` extension. Docusaurus will automatically remove the `.md` extension from the URL. The file path is needed for Docuasurus to generate the correct URL for the page when versioning is enabled.
 
-The url of a page will be composed from the path of the file relative to the `content` directory.
 
-**Example** docs/content/1-introduction/1-what-is.md will have http://localhost:9000/introduction/what-is as the url
 
-In markdown you can reference this page relatively to the root of the domain using this syntax:
+The following example shows how to reference a page in various scenarios. Assume you have the following folder structure when reviewing the examples below:
+
+```shell
+.
+└── docs
+    └── docs-content
+        ├── architecture
+        │   ├── grpc.md
+        │   └── ip-addresses.md
+        ├── aws
+        │   └── iam-permissions.md
+        ├── clusters
+        └── security.md
+```
+
+#### Same Folder
+
+If you want to link to a file in the same folder, you can use the following syntax:
+
+```md
+![Insert a description here](name_of_file.md)
+```
+
+Because the file is in the same folder, you do not need to specify the path to the file. Docusaurus will automatically search the current folder for the file when compiling the markdown content.
+
+So, if you are in the file `grpc.md` and want to reference the file `ip-addresses.md`, you would use the following syntax:
+
+```md
+![A list of all Palette public IP addresses](ip-addresses.md)
+```
+
+#### Different Folder
+
+If you want to link to a file in a different folder, you have to specify the path to the file from where the current markdown file is located. 
+
+If you are in the file `security.md` and want to reference the file `iam-permissions.md`, you have to use the following syntax:
 
 ```md
-[Go to introduction](/introduction/what-is)
+![A list of all required IAM permissions for Palette](aws/iam-permissions.md)
 ```
 
-You can also reference pages that reside in the document folder, such as index pages. An example is the Dev Engine index page `/docs/04.5-devx.md`. To reference the Dev Engine index page in a documentat page, referce the page by the title.
+If you are in the file `grpc.md` and want to reference the file `iam-permissions.md`, you have to use the following syntax:
 
 ```md
-[Go to Dev Enging](/devx)
+![A list of all required IAM permissions for Palette](../aws/iam-permissions.md)
 ```
+
+#### Exceptions
+
+As of Docusarus `2.4.1`, the ability to reference pages that belong to another plugin is not available. To work around this limitation, URL  reference path must be used. 
```

Comment:
```suggestion
As of Docusarus `2.4.1`, the ability to reference pages that belong to another plugin is unavailable. To work around this limitation, use a URL reference path. 
```

Reply:
@karl-cardenas-coding Is a reference URL the same as a relative URL? If so, we should stick with the common term. An example would be helpful would be helpful here.